### DISCUSSION TITLE
突然获取不到前端资源

### DISCUSSION THREAD

**Discussion by katsukako** [mirai-mamori/Sakurairo#838] [Q&A] [answered]
到昨天为止显示都正常，包括字体和图片，icon等等，但是今天这些都无法获取了。我的服务器在美国，版本是v2.6.3.1

**Comment by katsukako** [comment]
视觉资源路径 https://s.nmxc.ltd/sakurairo_vision/@2.6/

**Comment by KotoriK** [comment]
ping @bymoye

**Comment by katsukako** [comment]
现在图片，icon等都正常显示了，非常感谢！就是Noto Serif SC字体好像还是有问题，我再去看看是不是我的问题。
html里是		
`<link rel="stylesheet" href="https://cdn2.tianli0.top/fonts/css?family=Noto+Serif|Noto+Serif+SC|Noto+Sans+SC|Dela+Gothic+One|Fira+Code&display=swap" media="all">`

**Reply by KotoriK** [reply, marked as answer]
Google字体可以切换谷歌字体源，主题配置里有

**Reply by katsukako** [reply]
好的谢谢
